types/events.go

```diff
@@ -70,6 +71,20 @@ type Cpu struct {
 	Throttling Throttling `json:"throttling,omitempty"`
 }
 
+type Cpuset struct {
+	Cpus                  []uint16 `json:"cpus,omitempty"`
```

Comment:
```suggestion
	CPUs                  []uint16 `json:"cpus,omitempty"`
```

Not 100% sure if it should be lowercase or uppercase "s"